src/librustc/dep_graph/graph.rs

```diff
@@ -1037,19 +1056,19 @@ impl CurrentDepGraph {
         fingerprint: Fingerprint
     ) -> DepNodeIndex {
         debug_assert!(!self.node_to_node_index.contains_key(&dep_node));
-        self.intern_node(dep_node, edges, fingerprint).0
+        self.intern_node(dep_node, edges, fingerprint)
     }
 
     fn intern_node(
```

Comment:
can you add a comment above the `fn intern_node` signature, indicating what the `bool` in the return type represents?

Reply:
The `bool` is removed, so no =P